qtpy/QtDBus.py

```diff
@@ -8,18 +8,18 @@
 """Provides QtDBus classes and functions."""
 
 import sys
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError, QtBindingMissingModuleError
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtBindingsNotFoundError, QtBindingMissingModuleError, QtModuleNotFoundError
 
-if sys.platform == 'linux':
-    if PYQT5:
-        from PyQt5.QtDBus import *
-    elif PYQT6:
-        from PyQt6.QtDBus import *
-    elif PYSIDE2:
-        raise QtBindingMissingModuleError(name='QtDBus')
-    elif PYSIDE6:
+if PYQT5:
+    from PyQt5.QtDBus import *
+elif PYQT6:
+    from PyQt6.QtDBus import *
+elif PYSIDE2:
+    raise QtBindingMissingModuleError(name='QtDBus')
+elif PYSIDE6:
+    if sys.platform != 'win32':
         from PySide6.QtDBus import *
     else:
-        raise PythonQtError("No Qt bindings could be found")
+        raise QtModuleNotFoundError(name='QtDBus', msg='{name} does not exist on this operating system.')
 else:
-    raise PythonQtError('QtDBus does not exist on this operating system')
\ No newline at end of file
+    raise QtBindingsNotFoundError()
```

Comment:
There's 54 instances of `PythonQtError('No Qt bindings could be found')`, so `QtBindingsNotFoundError()` is just a direct, cleaner replacement for it.

Reply:
I think having a more specific error could be better indeed :+1: